File: Insertion_Sort.js
function insertionSort1(n, arr) {
    // Write your code here
    var last = arr[n-1];
   while (arr[--n - 1] >last) {
    arr[n] = arr[n - 1];

    console.log(...arr);
  }
arr[n] = last;
console.log(...arr)

}

File: InsertionSort.py
def insertionSort1(n, arr):
    last=arr[n-1]
    while n > 1 and arr[n-2] > last:
        arr[n-1] = arr[n-2]
        n -= 1
        print(*arr)
    arr[n-1] = last 
    print(*arr)  
